### PR TITLE
Add ability to specify order of loading fixtures

### PR DESCRIPTION
**Pull request by marcm-nix**
Might be useful in case of loading depending fixtures like groups and users

Fixture should implement `Facile\MongoDbBundle\Fixtures\OrderedFixtureInterface` with `getOrder` method

For reference: https://github.com/facile-it/mongodb-bundle/pull/53

### CODE REVIEW

File: src/Command/LoadFixturesCommand.php [hunk 1]
```diff
@@ -109,4 +113,36 @@ protected function loadPaths($paths)
...
+
+            if ($isFixture1Instance && $isFixture2Instance) {
+                return $fixture1->getOrder() - $fixture2->getOrder();
+            }
+
+            if (!$isFixture1Instance && !$isFixture2Instance) {
+                return 1;
+            }
+
+            return ($isFixture1Instance) ? -1 : 1;
```
[the remark this answers is not in the document]
Comment (marcm-nix) [reply src/Command/LoadFixturesCommand.php]: This is how original doctrine fixtures order works which was style inspired.
And this is how I'd see it if I prioritize half of my fixtures.

[the remark this answers is not in the document]
Comment (Jean85) [reply src/Command/LoadFixturesCommand.php]: I agree. If a fixture doesn't have the interface, it should mean that it doesn't need to be loaded in a specific order, so it's ok to load them first.

[the remark this answers is not in the document]
Comment (marcm-nix) [reply src/Command/LoadFixturesCommand.php]: Oh, I messed something up with checking doctrine's code.
After rethinking it you might be right, modified the code to prioritize fixtures without interface.

**Comment by ilario-pierbattista** [comment]
Ok to me, thanks @marcm-nix!

**Pull request merged**